master/buildbot/db/sourcestamps.py

```diff
@@ -166,7 +167,27 @@ def _rowToSsdict_thd(self, conn, row):
                 ssdict['patch_subdir'] = row.subdir
                 ssdict['patch_author'] = row.patch_author
                 ssdict['patch_comment'] = row.patch_comment
-                body = base64.b64decode(row.patch_base64)
+
```

Comment:
can you explain what you are trying to do, what the problem you are trying to fix?

Reply:
@tardyp there are some problems in `TestRealDB` : https://travis-ci.org/buildbot/buildbot/jobs/469199833#L1183

when tested against a real MySQL or Postgres database.

I was trying to chase these down.

Reply:
I found and fixed the issue here: https://github.com/buildbot/buildbot/pull/4500